.github/ISSUE_TEMPLATE/2_bug.md

```diff
@@ -16,36 +16,75 @@ assignees: ''
       * https://api.flutter.dev/
       * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
 
+     Issues directly asking for support will be closed.
+
      If you have found a bug or if our documentation doesn't have an answer
      to what you're looking for, then fill out the template below. Please read
      our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
-## Steps to Reproduce
+### Description
+
+<!--
+     Please describe in detail the problem you are running into.
+
+     List concrete steps to reproduce the problem, if this will better illustrate it:
+
+        1. ...
+        2. ...
+        3. ...
```

Comment:
@Hixie does it make sense to move it out of comment too (as above), or rather remove these lines at all?